doc/roadmap.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## 0.4.0
+
+- identity and access management
+- CRDT type schema support
+- fast redis backend support
+- query planner and optimizer
+- transaction monitoring
```

Comment:
does the coming release contain all these features? I saw the preparation for 0.4.0 already but didn't hear about CRDT nor Redis nor identity and access management. Change that already to mirror the coming release?

Reply:
I have removed the roadmap now entirely and added a note that we will be working with discussions now.